SPECS/cairo/cairo.spec

```diff
@@ -69,7 +69,8 @@ It contains the libraries and header files to create applications
         --enable-xlib-xrender=no \
         --enable-gobject \
         CFLAGS="-O3 -fPIC" \
```

Comment:
This looks out of place to me .. although i understand this is not a change you introduced ..

Reply:
True, I'm leaving it the way it is right now, to not destabilize the build too much.